scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
+
+import os
+import sys
+import pexpect
+
+arguments1 = {}
+arguments1["--scale"] = ".01"
+arguments1["--use_prepared_statements"] = "true"
+arguments1["--queries"] = "'1,13,19'"
+arguments1["--time"] = "1"
+arguments1["--runs"] = "100"
+arguments1["--mode"] = "'PermutedQuerySet'"
+arguments1["--encoding"] = "'Unencoded'"
+arguments1["--compression"] = "'SIMD-BP128'"
```

Comment:
I have no idea what you guys are doing here, but just looking at the config: compression does only affect a few encodings (dict, lz4, for FoR also but that one does only handle ints).
So for Unencoded there would be no effect.

Reply:
No specified encoding should default to dict in arguments2. So here the compression does have an effect.